source/local/local_test.go

```diff
@@ -33,3 +36,43 @@ func TestLocalSource(t *testing.T) {
 	assert.Empty(t, l)
 
 }
+
+func TestGetExpirationDate(t *testing.T) {
+	expectedFeaturesLen := 1
+	pwd, _ := os.Getwd()
+	featureFilesDir = filepath.Join(pwd, "testdata/features.d")
+
+	features, err := getFeaturesFromFiles()
+
+	assert.NoError(t, err)
+	assert.Equal(t, expectedFeaturesLen, len(features))
+}
+
+func TestParseComments(t *testing.T) {
+	testCases := []struct {
+		name        string
+		comment     string
+		expectedErr error
+	}{
+		{
+			name:        "valid comment",
+			comment:     "# +expiry-time=2080-07-28T11:22:33Z",
+			expectedErr: nil,
+		},
+		{
+			name:        "unknown comment",
+			comment:     "# +random-key=random-value",
+			expectedErr: ErrorUnknownDirective,
+		},
```

Comment:
For the `# + something` I added it. However, the comment `# Notes:` would not even pass to the `parseDirectives` function, because we are doing a filter before parsing directives, so general comments are excluded. Therefore, I added another file in the testdata folder that has the `# Notes:` comment.